src/index.ts

```diff
@@ -20,12 +20,12 @@ type LazyNamed = (
   thenable: () => Promise<{
     [name: string]: any;
   }>,
-  name: string
+  name: string = 'default'
 ) => React.LazyExoticComponent<React.ComponentType<any>>;
 
 /**
  * @param {() => Promise<*>} thenable
- * @param {String} name
+ * @param {String} [name="default"]
```

Comment:
This doesn't actually seem to affect the type checking aspect as long as there is a predefined type above